src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Tags/ShapePagerTag.cs

```diff
@@ -1,75 +1,72 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 using System.Text.Encodings.Web;
 using System.Threading.Tasks;
 using Fluid;
 using Fluid.Ast;
 using Fluid.Values;
 using Newtonsoft.Json;
 using OrchardCore.DisplayManagement.Shapes;
-using OrchardCore.Liquid.Ast;
 using OrchardCore.Mvc.Utilities;
 
 namespace OrchardCore.DisplayManagement.Liquid.Tags
 {
-    public class ShapePagerTag : ExpressionArgumentsTag
+    public class ShapePagerTag
     {
         private static readonly HashSet<string> _properties = new HashSet<string>
         {
             "Id", "PreviousText", "NextText", "PreviousClass", "NextClass", "TagName", "ItemTagName"
```

Comment:
Would it make sense to also have the expected "raw" before conversion using `ToPascalCaseUnderscore` value here to match without need to always evaluate name with that function inside the foreach?